src/core/server/graph/schema/schema.graphql

```diff
@@ -1514,6 +1514,18 @@ type StaffConfiguration {
   label is the string used when displaying the STAFF badge to users.
   """
   label: String!
+
+  """
+  moderatorLabel is the string used when displaying the MODERATOR badge to users. Defaults
+  to value of `staff.label`
+  """
+  moderatorLabel: String
+
+  """
+  adminLabel is the string used when displaying the ADMIN badge to users. Defaults
+  to value of `staff.label`
+  """
+  adminLabel: String
```

Comment:
These should be `String!` and not `String`.

Reply:
Remove the notice about the default value as well because that's applied only until the first save is hit.